integration/kafka/alluxio-worker-standalone-example.properties

```diff
@@ -0,0 +1,42 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the “License”). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+# Sample configuration for a standalone Kafka Connect worker that uses Avro serialization and
+# integrates the the Schema Registry. This sample configuration assumes a local installation of
+# Confluent Platform with all services running on their default ports.
+
+# Bootstrap Kafka servers. If multiple servers are specified, they should be comma-separated.
+bootstrap.servers=localhost:9092
+
+# The converters specify the format of data in Kafka and how to translate it into Connect data.
+# Every Connect user will need to configure these based on the format they want their data in
+# when loaded from or stored into Kafka
+
+key.converter=io.confluent.connect.avro.AvroConverter
+key.converter.schema.registry.url=http://localhost:8081
+value.converter=io.confluent.connect.avro.AvroConverter
+value.converter.schema.registry.url=http://localhost:8081
+
+#key.converter=org.apache.kafka.connect.json.JsonConverter
```

Comment:
If we ends up putting the JSON converters in the comments, we should at least add some description on why we need it.